### PR TITLE
Nace label three

### PR DESCRIPTION
**Pull request by Tom-Nagy**
# Business Analysis 🟠 Code Institute 🟠 Community Project - Pull Request 🎉🎉

## Pull Request References:
    - #12   

## PR contains:
- [x] Add the third level chart display: route and template

Further down the lane will need to add the work on the linking them together with templates update.

## Breaking changes
- 'None'

## Screenshots/Gifs/Media
nace-sub-cat-chart

### CODE REVIEW

**Comment by stefdworschak** [comment]
What's the difference between this PR and https://github.com/Code-Institute-Community/business-analysis-project/pull/50?

**Comment by Tom-Nagy** [comment]
The difference with PR #50 is that it provides a chart for sub-categories and #50 is for categories.

**Comment by Tom-Nagy** [comment]
So I have made all the changes about naming, and renamed the files accordingly.
I believe that merging Nace label three is enough as its the latest version of the three PR they are build up on the previous version.
I hope that is ok.
If there would have been a better way to do it let me know please. (maybe i just should have waited!)

**Comment by auxfuse** [comment]
Hi @Tom-Nagy we have a merge conflict present on this PR in the `requirements.txt` file. Can you resolve and then push back to the branch that is associated with this PR to update same and then we can review again? 🤜

**Comment by Tom-Nagy** [comment]
I have deleted the requireements.txt file and somehow i cannot click on the Resolve conflicts button at the moment.
Let me know if i need to make some more changes.

**Comment by Tom-Nagy** [comment]
@auxfuse I did a rebase on my branch (fun experience) and integrated my branch in the main project that way there is no more conflicts and everything should be good to go.
I hope i did not delete any relevant commit history, but i don't think so.

File: app/templates/charts/nace_one_chart.html [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+                scales: {
+                    y: {
+                        beginAtZero: true
+                    }
+                }
+            }
+        });
+    </script>
+
+{% endblock %}
```
Comment (auxfuse) [inline app/templates/charts/nace_one_chart.html:42]: Hi @Tom-Nagy can you add in a Newline at the End of this file please? 😄

File: app/templates/charts/nace_three_chart.html [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+                scales: {
+                    y: {
+                        beginAtZero: true
+                    }
+                }
+            }
+        });
+    </script>
+
+{% endblock %}
```
Comment (auxfuse) [inline app/templates/charts/nace_three_chart.html:47]: Newline at End of File needed here please.

File: app/templates/charts/nace_two_chart.html [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+                scales: {
+                    y: {
+                        beginAtZero: true
+                    }
+                }
+            }
+        });
+    </script>
+
+{% endblock %}
```
Comment (auxfuse) [inline app/templates/charts/nace_two_chart.html:47]: Newline at End of File needed here please.

**Review by auxfuse — CHANGES_REQUESTED** [review]
Hi @Tom-Nagy Just a few changes to be made here for Good Code Quality practices. Can you make the changes as requested please?

File: app/templates/charts/nace_one_chart.html (hunk 1 above)
Comment (Tom-Nagy) [reply app/templates/charts/nace_one_chart.html]: done

File: app/templates/charts/nace_three_chart.html (hunk 2 above)
Comment (Tom-Nagy) [reply app/templates/charts/nace_three_chart.html]: done

**Comment by Tom-Nagy** [comment]
All done :)

**Review by auxfuse — APPROVED** [review]
@Tom-Nagy Great work! 🤜 Really looking forward to seeing these Graphs in play. 🎉

**Pull request merged**